tests/mgbench/Dockerfile.mgbench_bolt_client

```diff
@@ -0,0 +1,41 @@
+FROM ubuntu:22.04 AS mg_bench_client_build_base
+
+SHELL ["/bin/bash", "-c"]
+ARG TOOLCHAIN_VERSION
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+USER root
+
+RUN apt update && apt install -y \
+    ca-certificates wget git
+
+RUN wget -q https://s3-eu-west-1.amazonaws.com/deps.memgraph.io/${TOOLCHAIN_VERSION}/${TOOLCHAIN_VERSION}-binaries-ubuntu-22.04-amd64.tar.gz \
+    -O ${TOOLCHAIN_VERSION}-binaries-ubuntu-22.04-amd64.tar.gz \
```

Comment:
Env var or flag, `amd64` | `arm64` (only Ubuntu 22.04) naming

Reply:
This has been solved.